components/homme/cmake/HommeMacros.cmake

```diff
@@ -156,7 +156,7 @@ macro(createTestExec execName execType macroNP macroNC
   ENDIF ()
 
   IF (HOMME_USE_KOKKOS)
-    target_link_libraries(${execName} kokkos)
+    target_link_libraries(${execName} Kokkos::kokkos)
```

Comment:
These mods in hommd, once merged upstream, may cause issues with the HOMME/HOMMEBFB test cases. We should verify that that's not the case.

Reply:
In other words, you could do these changes in the upstream, and see if HOMME/HOMMEBFB pass (on chrysalis)

Reply:
It should work as long as homme is managing kokkos in the modern way.